### PR TITLE
Updates to swagger documentation

### LINKED ISSUE #2635: Swagger errors
- Gitea version (or commit ref): 1ad902d5298202d5be14fd5a9c8ed6ce781a23c8
- Git version:
- Operating system:
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

There are errors in our swagger config, see https://online.swagger.io/validator/debug?url=https://try.gitea.io/swagger.v1.json

## Screenshots
image

### PR DESCRIPTION
**Pull request by ethantkoenig**
This is part of a larger updating/refactoring endeavor that will fix https://github.com/go-gitea/gitea/issues/2635 and other problems with our swagger docs. See https://github.com/ethantkoenig/gitea/tree/swagger if you're curious.

This PR has no functional change.

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
Please send a PR to update this to gitea vendor.

File: gitea/release.go [hunk 1]
```diff
@@ -13,20 +13,21 @@ import (
...
-	Note         string        `json:"body"`
-	URL          string        `json:"url"`
-	TarURL       string        `json:"tarball_url"`
-	ZipURL       string        `json:"zipball_url"`
-	IsDraft      bool          `json:"draft"`
-	IsPrerelease bool          `json:"prerelease"`
-	CreatedAt    time.Time     `json:"created_at"`
-	PublishedAt  time.Time     `json:"published_at"`
-	Publisher    *User         `json:"author"`
-	Attachments  []*Attachment `json:"assets"`
```
Comment (lunny) [inline gitea/release.go:29]: Yes, `attachements` should not be removed here.

Comment (ethantkoenig) [reply gitea/release.go:29]: See discussion in #83

Comment (ethantkoenig) [reply gitea/release.go:29]: I probably should have commented in this PR about why I removed it, so apologies for miscommunication on my part.

**Pull request merged**